src/doc/src/reference/specifying-dependencies.md

```diff
@@ -19,18 +19,32 @@ guide](../guide/index.md), we specified a dependency on the `time` crate:
 time = "0.1.12"
 ```
 
-The string `"0.1.12"` is a version requirement. Although it looks like a
-specific *version* of the `time` crate, it actually specifies a *range* of
-versions and allows [SemVer] compatible updates. An update is allowed if the new
-version number does not modify the left-most non-zero digit in the major, minor,
-patch grouping. In this case, if we ran `cargo update -p time`, cargo should
-update us to version `0.1.13` if it is the latest `0.1.z` release, but would not
-update us to `0.2.0`. If instead we had specified the version string as `1.0`,
-cargo should update to `1.1` if it is the latest `1.y` release, but not `2.0`.
-The version `0.0.x` is not considered compatible with any other version.
+As specified in [SemVer]
+a version identifier is composed of up to three integers separated by
+a period: `x`, `x.y`, or `x.y.z` where x is the major version, y is the minor
+version and z is the patch version.
```

Comment:
This is talking about "version identifier" which sounds like its talking about "versions" but the old documentation very clearly and consistently used the term "version requirement" and specifically called out that version requirements are different from versions.

While the following paragraph tries to resolve some of this confusion, it will be too late for the reader.

Reply:
I understand your critique and will incorporate some changes.

Here is what motivates the new structure:

As a new reader/new to Rus: I can tell you that whether the term was "version requirement" or "version identifier" made zero difference. I was focused on the thing that was instantly presented, which is/was a string of the form x.y.z

Indeed, the text you're referencing begins with "As specified in [SemVer]" and discusses what a version *identifier* is defined as therein. I am providing the baseline definition/understanding of what a SemVer version identifier looks like and what its components (the x, y & z) are, so that the reader has a baseline.

I then go on to specify that such a bare identifier appearing in a toml is in fact not purely a specific version identifier, but is instead interpreted as a "caret requirement." In fact, the old text left me scratching my head because it suddenly talked about "an alternative syntax for the default strategy" ... what "default strategy"?

Reply:
> As a new reader/new to Rust: I can tell you that whether the term was "version requirement" or "version identifier" made zero difference. I was focused on the thing that was instantly presented, which is/was a string of the form x.y.z

For your case yes.  Maybe not for other cases.

We also have to deal with people upgrading their knowledge.  Conflating topics will just lead to more, overall, confusion,

> As a new reader/new to Rust: I can tell you that whether the term was "version requirement" or "version identifier" made zero difference. I was focused on the thing that was instantly presented, which is/was a string of the form x.y.z

semver makes no reference to version requirements and versions must declare all fields according to semver, so referencing back to semver is incorrect.

Reply:
> One of the "improvements" to the original text is that the meaning of x, y & z then remain consistent (x is always a major version).

I have a feeling that us going through an edit process with all of these different changes in the same PR will require a lot of back and forth and take a lot of time on both of our sides.

I would ask that you close this PR and instead create one PR per change, like the one quoted above
- Make less controversial changes faster to get in as they don't get blocked on other changes being reviewed
- Reduce the chance of new issues being opened in each review pass from the original change, leading to extra frustration and time
- Reduce reviewer burden which is [limited at the moment](https://github.com/rust-lang/cargo/blob/master/CONTRIBUTING.md) by making it easier for reviewers to identify changes

https://github.com/rust-lang/cargo/pull/11082/commits is an example of this.  While I did it in 1 PR due to my confidence in the changes, I split it up to make it easier for my reviewers

Reply:
Ed:

> semver makes no reference to version requirements

The very first thing on the semver page, parroted in my suggested edit:

```
Given a version number MAJOR.MINOR.PATCH, increment the:

MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backwards compatible manner
PATCH version when you make backwards compatible bug fixes
Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format
```

Go ahead and just close the PR. This PR is not worth the level resistance to the changes.

A.

Reply:
> However, what I was defining is a semver version identifier, which it is then subsequently stated is interpreted as a caret requirement.

Except as described (include "**up to** three integers"), no such thing exists.  Just versions (3 required fields) and version requirements (optional operator with 0-3 fields)

Documentation changes can be some of the hardest to make because they require being both technically correct enough not to one subset of people while easy enough to onboard another subset of people.